.pre-commit-config.yaml

```diff
@@ -26,17 +25,6 @@ repos:
     hooks:
       - id: reorder-python-imports
 
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v0.761"
-    hooks:
-      - id: mypy
-
-  - repo: https://gitlab.com/pycqa/flake8
-    rev: "3.7.9"
-    hooks:
-      - id: flake8
-        args: ["--select=E9,F63,F7,F82"]
```

Comment:
I have used this config multiple times to catch bugs

Reply:
is it doing anything that mypy wouldn't? we have tons of test files with intentionally unused variables